client/src/views/Admin.vue

```diff
@@ -53,10 +66,41 @@ import { patchConfiguration } from '../api/api';
           label="Show Sequence Annotations"
         />
       </v-row>
+      <v-row>
+        <v-switch
+          v-model="settings.runInferenceOnUpload"
+          :color="settings.runInferenceOnUpload ? 'primary' : ''"
+          label="Run Inference on Upload"
+        />
+      </v-row>
+      <v-row>
+        <v-col cols="3">
+          <v-text-field
+            v-model="settings.spectrogramXStretch"
+            density="compact"
+            type="number"
+            step="0.25"
+            min="1"
+            max="5"
+            label="Stretch spectrogram"
+          />
```

Comment:
I don't believe v-text-fields support min/max.  I'm able to set it to negative numbers (the interface just defaults to the 1x when the config has a negative number).  Probably just swap it to a v-slider form 0-10X with the step of 0.25

Reply:
See d0bc693